### ISSUE TITLE
Unlabeled buttons after each game in the library?

### ISSUE THREAD

**Issue by AERobert** [angstsmurf/spatterlight#81]
In the library, there are buttons after each game entry that are not labeled clearly in VoiceOver and don't seem to have any effect (I thought they might be play or delete buttons, but it seems not) Specifically, VO just calls them "♡ button".

**Comment by angstsmurf** [comment]
This is the "Like" button, which corresponds to the "star" in Apple's Music app (but with a heart symbol instead of a star). The "Like" column can be moved and hidden, by the way, although I'm not sure how easy that is to do using VoiceOver. I've added some descriptive accessibility labels which should make it a little clearer, but VoiceOver will often still say "heart suit" before reading them.

Implemented in test build 1.2.1 here:
https://github.com/angstsmurf/spatterlight/releases/tag/v1.2.1b

**Comment by AERobert** [comment]
The new label is nice. I will at least know what the buttons are for now.

**Comment by angstsmurf** [comment]
Right. I think we can close this, then.
